nncf/experimental/tensor/functions.py

```diff
@@ -0,0 +1,362 @@
+        )
+    )
+
+
+@functools.singledispatch
+@_tensor_guard
+def any(a: TTensor, axis: Optional[Union[int, Tuple[int]]] = None) -> TTensor:  # pylint: disable=redefined-builtin
+    """
+    Test whether any tensor elements along a given axis evaluate to True.
+
+    :param a: The input tensor.
+    :param axis: Axis or axes along which a logical OR reduction is performed.
+    :return: A new boolean or tensor is returned unless out is specified,
+      in which case a reference to out is returned.
+    """
+    return Tensor(any(a.data, axis))
+
+
+@functools.singledispatch
+@_tensor_guard
+def count_nonzero(a: TTensor, axis: Optional[Union[int, Tuple[int]]] = None) -> TTensor:
+    """
+    Counts the number of non-zero values in the tensor input.
+
+    :param a: The tensor for which to count non-zeros.
+    :param axis: Axis or tuple of axes along which to count non-zeros.
+    :return: Number of non-zero values in the tensor along a given axis.
+      Otherwise, the total number of non-zero values in the tensor is returned.
+    """
+    return Tensor(count_nonzero(a.data, axis))
+
+
+@functools.singledispatch
+@_tensor_guard
+def isempty(a: TTensor) -> bool:
+    """
+    Return True if input tensor is empty.
+
+    :param a: The input tensor.
+    :return: True is tensor is empty, otherwise False.
+    """
+    return Tensor(isempty(a.data))
+
+
+@functools.singledispatch
+@_tensor_guard
+def isclose(a: TTensor, b: TTensor, rtol: float = 1e-05, atol: float = 1e-08, equal_nan: bool = False) -> TTensor:
+    """
+    Returns a boolean array where two arrays are element-wise equal within a tolerance.
+
+    :param a: The first input tensor.
+    :param b: The second input tensor.
+    :param rtol: The relative tolerance parameter, defaults to 1e-05.
+    :param atol: The absolute tolerance parameter, defaults to 1e-08.
+    :param equal_nan: Whether to compare NaN`s as equal. If True,
+      NaN`s in a will be considered equal to NaN`s in b in the output array.
+      Defaults to False.
+    :return: Returns a boolean tensor of where a and b are equal within the given tolerance.
+    """
+    return Tensor(
+        isclose(
+            a.data,
+            unwrap_tensor_data(b),
+            rtol=rtol,
+            atol=atol,
+            equal_nan=equal_nan,
+        )
+    )
+
+
+@functools.singledispatch
+@_tensor_guard
+def maximum(x1: TTensor, x2: TTensor) -> TTensor:
+    """
+    Element-wise maximum of tensor elements.
+
+    :param x1: The first input tensor.
+    :param x2: The second input tensor.
+    :return: Output tensor.
+    """
+    return Tensor(maximum(x1.data, unwrap_tensor_data(x2)))
+
+
+@functools.singledispatch
+@_tensor_guard
+def minimum(x1: TTensor, x2: TTensor) -> TTensor:
+    """
+    Element-wise minimum of tensor elements.
+
+    :param x1: The first input tensor.
+    :param x2: The second input tensor.
+    :return: Output tensor.
+    """
+    return Tensor(minimum(x1.data, unwrap_tensor_data(x2)))
```

Comment:
Why not do like this?
```suggestion
    return Tensor(minimum(x1.data, x2.data))
```

Reply:
To call function like 
```python
import torch
from nncf.experimental.tensor import Tensor, functions
unctions.minimum(Tensor(torch.tensor([1,2])), 0)    # nncf.Tensor(tensor([0, 0]))
```